test_tools/sigma_verify_rules.py

```diff
@@ -17,84 +17,21 @@
 It also does not require you to have a full blown Timesketch instance.
 Default this tool will show only the rules that cause problems.
 Example way of running the tool:
-  $ sigma_verify_rules.py --config_file ../data/sigma_config.yaml ../data/linux/
+$ PYTHONPATH=. python3 test_tools/sigma_verify_rules.py --config_file
```

Comment:
I would say that this should be 

```shell
cd test_tools
PYTHONPATH=. python3 test_tools/sigma_verify_rules.py --config......
...
```

that is you run it inside the `test_tools` folder, otherwise you are using the pythonpath as the root directory, which means you use the `actual` timesketch files, instead of the ones in the test folder.

Reply:
so the idea here is to actually use the real timesketch files (the util file) otherwise I would need to copy the file over and always keep it sync.